src/view/table/Rows.svelte

```diff
@@ -0,0 +1,161 @@
+				duration: 200,
+				id: 'mmc-package-menu',
+				x: event.pageX,
+				y: event.pageY,
+				items
+			});
+		}
+	}
+</script>
+
+<tbody>
+	{#each [...$spreadsheetStore] as row, i (row.id)}
+		<tr
+			animate:flip={{ duration: 250 }}
+			on:contextmenu={(event) => onContextMenu(event, row.id)}
+			data-status={row.status}
+			title={statusData[row.status].explanation}
+		>
+			<td>{row.title}</td>
+			{#if $details}
+				<td>{row.type}</td>
+				<td>{row.id}</td>
+				<td class="center">{row.version}</td>
+				<td class="center">{row.status}</td>
+			{/if}
+			<td>{row.notes}</td>
+		</tr>
+	{/each}
+</tbody>
+
+<style lang=scss>
+	tbody {
+		overflow: scroll;
+
+		tr {
+			transition: background-color 250ms;
+
+			&:hover {
+				box-shadow: inset 0 8px 6px -6px rgba(0, 0, 0, 0.5), inset 0 -8px 6px -6px rgba(0, 0, 0, 0.5);
+				box-sizing: border-box;
+			}
+
+			&:nth-child(odd) {
+				&[data-status="X"] {
+					background-color: hsla(0, 100%, 60%, 88%);
+					&:hover { background-color: hsla(0, 100%, 58%, 92.4%); }
+				}
+				&[data-status="O"] {
+					background-color: hsla(45, 90%, 60%, 88%);
+					&:hover { background-color: hsla(45, 90%, 58%, 92.4%); }
+				}
+				&[data-status="B"] {
+					background-color: hsla(30, 90%, 40%, 88%);
+					&:hover { background-color: hsla(30, 90%, 38%, 92.4%); }
+				}
+				&[data-status="G"] {
+					background-color: hsla(120, 40%, 50%, 88%);
+					&:hover { background-color: hsla(120, 40%, 48%, 92.4%); }
+				}
+				&[data-status="N"] {
+					background-color: hsla(200, 60%, 50%, 88%);
+					&:hover { background-color: hsla(200, 60%, 48%, 92.4%); }
+				}
+				&[data-status="A"] {
+					background-color: hsla(0, 0%, 50%, 88%);
+					&:hover { background-color: hsla(0, 0%, 48%, 92.4%); }
+				}
+				&[data-status="U"] {
+					background-color: hsla(0, 0%, 100%, 88%);
+					&:hover { background-color: hsla(0, 0%, 98%, 92.4%); }
+				}
+			}
+
+			&:nth-child(even) {
+				&[data-status="X"] {
+					background-color: hsla(0, 100%, 61.4%, 70.4%);
+					&:hover { background-color: hsla(0, 100%, 64.2%, 66%); }
+				}
+				&[data-status="O"] {
+					background-color: hsla(45, 90%, 61.4%, 70.4%);
+					&:hover { background-color: hsla(45, 90%, 64.2%, 66%); }
+				}
+				&[data-status="B"] {
+					background-color: hsla(30, 90%, 41.4%, 70.4%);
+					&:hover { background-color: hsla(30, 90%, 44.2%, 66%); }
+				}
+				&[data-status="G"] {
+					background-color: hsla(120, 40%, 51.4%, 70.4%);
+					&:hover { background-color: hsla(120, 40%, 54.2%, 66%); }
+				}
+				&[data-status="N"] {
+					background-color: hsla(200, 60%, 51.4%, 70.4%);
+					&:hover { background-color: hsla(200, 60%, 54.2%, 66%); }
+				}
+				&[data-status="A"] {
+					background-color: hsla(0, 0%, 51.4%, 70.4%);
+					&:hover { background-color: hsla(0, 0%, 54.2%, 66%); }
+				}
+				&[data-status="U"] {
+					background-color: hsla(0, 0%, 101.4%, 70.4%);
+					&:hover { background-color: hsla(0, 0%, 104.2%, 66%); }
+				}
+			}
```

Comment:
Using SCSS for this is definitely cleaner than vanilla CSS and better for performance than JS, I kind of wish there was a way to do this while still pulling the data from one single source i.e. `src/store/statusData.js` which should be the single source of those HSL values ideally

Reply:
Yeah.. There was something funky w/ the programmatic version and generating. I probably spent ~3-4 hours at least trying to get the table colors to look right / better first taking a look at what you tried then eventually settling on the Sass route as it was going to be dependable and do exactly what is encoded in the above snippet. I actually used your code to generate that Sass... No way by hand I decided on those values.  Here is the snippet that generated that code which was modified from your initial effort: 

```
	const statusColors = {
		X: { hsl: [0, 100, 60] },
		O: { hsl: [45, 90, 60] },
		B: { hsl: [30, 90, 40] },
		G: { hsl: [120, 40, 50] },
		N: { hsl: [200, 60, 50] },
		A: { hsl: [0, 0, 50] },
		U: { hsl: [0, 0, 100] }
	};
// 30 / 50 initially
	function logStyle() {

let text = '&:nth-child(odd) {\n';
		for (const [status, data] of Object.entries(statusColors))
		{
			const { hsl } = data;

text += `	&[data-status="${status}"] {
		background-color: hsla(${hsl[0]}, ${hsl[1]}%, ${hsl[2] + 0}%, ${22 * 4}%);
		&:hover { background-color: hsla(${hsl[0]}, ${hsl[1]}%, ${hsl[2] - 2}%, ${22 * 4.2}%); }
	}\n`;
		}
text += '}\n\n';

text += '	&:nth-child(even) {\n';
		for (const [status, data] of Object.entries(statusColors))
		{
			const { hsl } = data;

text += `	&[data-status="${status}"] {
		background-color: hsla(${hsl[0]}, ${hsl[1]}%, ${hsl[2] + 7 * 0.2}%, ${22 * 3.2}%);
		&:hover { background-color: hsla(${hsl[0]}, ${hsl[1]}%, ${hsl[2] + 7 * 0.6}%, ${22 * 3}%); }
	}\n`;
		}
text += '}';

console.log(text);
	}

	logStyle();
```

Mind that the above code is ugly / quick hack to generate the Sass. What I have above is tweaked from your initial values as well. You had values well above the thresholds for `hsla` colors and even some that I generated are, but not by much. 

What made the table look much better was adding the inset box shadow when hovering which is just above the Sass block you quoted:
```
			&:hover {
				box-shadow: inset 0 8px 6px -6px rgba(0, 0, 0, 0.5), inset 0 -8px 6px -6px rgba(0, 0, 0, 0.5);
				box-sizing: border-box;
			}
```